packages/@react-spectrum/dialog/test/DialogContainer.test.js

```diff
@@ -43,6 +43,7 @@ describe('DialogContainer', function () {
     );
 
     let button = getByRole('button');
+    act(() => button.focus());
```

Comment:
unneeded i'm pretty sure, why was it added?

Reply:
I'm not so sure about that. `triggerPress` only seems to fire `mouseDown`, `mouseUp` and `click`, I don't think it forces focus. See 
https://github.com/adobe/react-spectrum/blob/418e59ba5be2418cb52ea5087162a6faeeb1cc5a/packages/dev/test-utils/src/events.ts#L19-L23